pkg/kubelet/rkt/rkt.go

```diff
@@ -1887,15 +1886,29 @@ func (r *Runtime) PortForward(pod *kubecontainer.Pod, port uint16, stream io.Rea
 	if lookupErr != nil {
 		return fmt.Errorf("unable to do port forwarding: socat not found.")
 	}
-
-	args := []string{"-t", fmt.Sprintf("%d", listResp.Pods[0].Pid), "-n", socatPath, "-", fmt.Sprintf("TCP4:localhost:%d", port)}
-
 	nsenterPath, lookupErr := exec.LookPath("nsenter")
 	if lookupErr != nil {
 		return fmt.Errorf("unable to do port forwarding: nsenter not found.")
 	}
 
-	command := exec.Command(nsenterPath, args...)
+	var args []string
+	if r.config.Stage1Image != "stage1-kvm.aci" {
+		args = append(args, "-t", fmt.Sprintf("%d", listResp.Pods[0].Pid), "-n")
+	}
+
+	if r.config.Stage1Image == "stage1-kvm.aci" {
+		args = append(args, "-", fmt.Sprintf("TCP4:%s:%d", listResp.Pods[0].GetNetworks()[1].Ipv4, port))
+	} else {
+		args = append(args, socatPath, "-", fmt.Sprintf("TCP4:localhost:%d", port))
+	}
+
+	command := new(exec.Cmd)
```

Comment:
move this line just after `var args` and then combine all 3 ifs into single one.